src/components/user-steps-wrapper/UserStepsWrapper.tsx

```diff
@@ -29,10 +29,10 @@ const UserStepsWrapper: React.FC<UserStepsWrapperProps> = ({ userRole }) => {
   }, [dispatch])
 
   const childrenArr = [
-    <GeneralInfoStep key='1' />,
-    <SubjectsStep key='2' />,
-    <LanguageStep key='3' />,
-    <AddPhotoStep key='4' />
+    <GeneralInfoStep btnsBox={undefined} key='1' />,
+    <SubjectsStep btnsBox={undefined} key='2' />,
+    <LanguageStep btnsBox={undefined} key='3' />,
+    <AddPhotoStep btnsBox={undefined} key='4' />
```

Comment:
```suggestion
    <GeneralInfoStep key='1' />,
    <SubjectsStep key='2' />,
    <LanguageStep key='3' />,
    <AddPhotoStep key='4' />
```

It's not good to throw ```undefined```. If you set props with ? ( like in SubjectStep component you did ) it means that it's unneccessary to throw props and you can just not pass them.

Reply:
Already fixed, thanks for suggestion